Identity/Identity.csproj

```diff
@@ -7,8 +7,8 @@
     <Company>Kontrol S.A.S.</Company>
     <Product>Tanker</Product>
     <Description>Small library for Tanker Identity generation</Description>
-    <AssemblyVersion>1.1.0</AssemblyVersion>
-    <Copyright>Copyright ©  2019</Copyright>
+    <AssemblyVersion>1.2.0</AssemblyVersion>
+    <Copyright>Copyright ©  2020</Copyright>
```

Comment:
Keep `2019`, or change to `Copyright © 2019-2020`. The whole point of copyright is to reference the initial year as a priority date!

Also remove the double space 😉